src/pages/Result.js

```diff
@@ -0,0 +1,27 @@
+import { useQuiz } from "../context/quiz-context";
+import "./Result.css";
+import { Results } from "../components";
```

Comment:
suggestion- You can use absolute paths for imports, these small changes can make a whole lot of difference in one's code quality and it will enhance your developer experience as well.

https://stackoverflow.com/questions/41993461/import-react-components-with-absolute-path
you can use the above link for reference if you intend to use absolute paths.

Reply:
will read about this and do this as an update to the code